PyMca5/PyMcaGui/io/TiledDataChannelTable.py

```diff
@@ -0,0 +1,191 @@
+from PyQt5 import QtWidgets
+
+from PyMca5.PyMcaGui import PyMcaQt as qt
+from PyMca5.PyMcaGui.io.SpecFileCntTable import CheckBoxItem
+
+class TiledDataChannelTable(qt.QTableWidget):
+    """
+    Creates the data channel table (second table) inside the QTiledWidget.
+    The selections for the x and y set the axes in the scan window.
+    """
+    sigTiledDataChannelTableSignal = qt.pyqtSignal(object)
+    def __init__(self, parent=None):
```

Comment:
[Formatting for clarity]

```suggestion
    sigTiledDataChannelTableSignal = qt.pyqtSignal(object)
    
    def __init__(self, parent=None):
```